interface/reports/patient_edu_web_lookup.php

```diff
@@ -43,10 +33,7 @@
 
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-3-2/index.js"></script>
-
```

Comment:
Hi Brady I added line 37, But When I unit tested, But the result is  the code on line 51 
 <?php Header::setupHeader(); ?>
 37 <title><?php echo xlt('Patient Education'); ?></title>
 38 <script type="text/javascript">
 39   function searchResultsPopup(search_term,link) {
 40     link_formatted = link.replace("[%]",encodeURIComponent(search_term));
 41     top.restoreSession();
 42     window.open(link_formatted);
 43   }
 44 </script>
 45
 46 </head>
 47
 48 <body class="body_top" onload="javascripts:document.forms[0].form_import_data.focus()">
 49 <div class="container">
 50   <div class="row">
 51     <div class='title'><?php echo  xlt('Web Search'); ?> - <?php echo xlt('Patient Education Materials'); ?></div>
 52   </div>
 53 </div>